frontend/__tests__/src/components/Metrics/MetricsStepper/MetricsStepper.test.tsx

```diff
@@ -258,4 +283,66 @@ describe('MetricsStepper', () => {
 
     expect(exportToJsonFile).toHaveBeenCalledWith(expectedFileName, expectedJson)
   })
+
+  it('should export json file when click save button in metrics page when all content is empty', async () => {
```

Comment:
`when all content is empty'` -> `given all content is empty'`

Reply:
same issue with the below test case

Reply:
have fixed this problem these two test case